db-facilities/pyproject 2.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
```

Comment:
@athursland looks like you might have accidentally added back the poetry configs?